.github/workflows/sanitizer.yml

```diff
@@ -32,6 +32,21 @@ jobs:
         run: |
           ./build-asan/uv_run_tests_a
 
+  sanitizers-linux:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup
+        run: |
+          sudo apt-get install ninja-build
+      - name: Envinfo
+        run: npx envinfo
+
+      # MSAN fails on newer kernels due to a bigger PIE slide than it can handle
+      - name: Disable ASLR
+        run: |
+          sudo sysctl -w kernel.randomize_va_space=0
```

Comment:
Note to self: setting vm.mmap_rnd_bits to some low value (8? 16?) may be sufficient. Maybe disabling/reducing ASLR will also fix ASAN on newer kernels.